autoload/neomake/makers/ft/text.vim

```diff
@@ -48,11 +48,11 @@ function! s:languagetool_maker.InitForJob(_jobinfo) abort
                 \ get(split(&spelllang, ','), 0,
                 \ s:getVar('neomake_text_languagetool_fallbackLanguage', 'auto') ) )
     " Optional Arguments
-    let motherTongue = s:getVar('neomake_text_languagetool_curl_motherTongue', '')
+    let motherTongue = s:getVar('neomake_text_languagetool_motherTongue', '')
     if !empty(motherTongue)
         let args += ['--motherTongue', motherTongue]
     endif
```

Comment:
E.g. here there is no need for a specific setting, just let users configure `--motherTongue` in args themselves.